### DISCUSSION TITLE
v0.1.0 - Initial Release

### DISCUSSION THREAD

**Discussion by Speyedr** [Speyedr/socialclub-notification-blocker#2] [Announcements]
Initial release.

You can find the download link to the official build under **Assets**.

If you encounter any issues while running the program, please either post about it in the discussion announcement or create an issue.

<hr /><em>This discussion was created from the release <a href='https://github.com/Speyedr/socialclub-notification-blocker/releases/tag/v0.1.0'>v0.1.0 - Initial Release</a>.</em>

**Comment by ChiggaFile** [comment]
im not receive any invites or spectator bug for past 30 min.
